webapp/src/components/AssetPage/ListingsTable/ListingsTable.tsx

```diff
@@ -0,0 +1,160 @@
+        })
+        .finally(() => setIsLoading(false))
+        .catch(error => {
+          console.error(error)
+        })
+    }
+  }, [asset, setIsLoading, setOrders, sortBy, page])
+
+  return (
+    <div className={styles.ListingsTable}>
+      {isLoading ? (
+        <div className={styles.emptyTable}>
+          <Loader active />
+        </div>
+      ) : orders.length === 0 ? (
+        <div className={styles.emptyTable}>
+          <span>{t('listings_table.there_are_no_listings')}</span>
+        </div>
+      ) : (
+        <>
+          <Table basic="very">
+            <Table.Header>
+              <Table.Row>
+                <Table.HeaderCell className={styles.headerMargin}>
+                  {t('listings_table.owner')}
+                </Table.HeaderCell>
+                <Table.HeaderCell>
+                  {t('listings_table.published_date')}
+                </Table.HeaderCell>
+                <Table.HeaderCell>
+                  {t('listings_table.expiration_date')}
+                </Table.HeaderCell>
+                <Table.HeaderCell>
+                  {t('listings_table.issue_number')}
+                </Table.HeaderCell>
+                <Table.HeaderCell>{t('listings_table.price')}</Table.HeaderCell>
+              </Table.Row>
+            </Table.Header>
+            <Table.Body className={isLoading ? 'is-loading' : ''}>
+              {orders?.map(order => (
+                <Table.Row key={order.id}>
+                  <Table.Cell>
+                    <LinkedProfile
+                      className={styles.linkedProfileRow}
+                      address={order.owner}
+                    />
+                  </Table.Cell>
+                  <Table.Cell>
+                    {getDateAndMonthName(order.createdAt)}
+                  </Table.Cell>
+                  <Table.Cell>
+                    {formatDistanceToNow(+order.expiresAt, {
+                      addSuffix: true
+                    })}
+                  </Table.Cell>
+                  <Table.Cell>
+                    <div className={styles.issuedIdContainer}>
+                      <div className={styles.row}>
+                        <span>
+                          <span className={styles.issuedId}>
+                            {order.tokenId}
+                          </span>
+                          / {total}
+                        </span>
+                      </div>
+                    </div>
+                  </Table.Cell>
+                  <Table.Cell>
+                    <div
+                      style={{
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        alignItems: 'center'
+                      }}
+                    >
+                      <Mana className="manaField">
+                        {formatWeiMANA(order.price)}
+                      </Mana>
+                      <Link
```

Comment:
can we add a check so we guarantee that the link will be valid?
```suggestion
                      {asset?.contractAddress ? asset<Link
```